README.md

```diff
@@ -61,13 +61,13 @@ Please feel free to flag any specific need you have need by creating an issue.
 
 Below are some features we have implemented to date:
 
-+ [Add, filter, sort, edit and track customers](#add-filter-sort-edit-and-track-customers)
++ [Add, filter, sort, edit, and track customers](#add-filter-sort-edit-and-track-customers)
```

Comment:
Hi @brandonyee-cs, I'm not a native english speaker but I believe the grammar was correct here: https://www.grammarly.com/blog/comma-before-and/

Reply:
Hey, regarding this grammar change, it is actually a widely debated comma when it comes to English. It is called the Oxford Comma, and it is generally considered to be grammatically correct. 

See here: https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/#:~:text=The%20Oxford%20(or%20serial)%20comma,use%20while%20others%20don't.